package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "my-vite-app",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint .",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@chakra-ui/icons": "^2.2.4",
```

Comment:
Why are we using two icon libraries: react-icons and @chakra-ui/icons?

Reply:
@chakra-ui/icons by default coming when we do npm i, react-icons I have installed because some icons are not there in
@chakra-ui/icons.